server/src/main/java/org/elasticsearch/health/HealthStatus.java

```diff
@@ -34,6 +41,15 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeByte(value);
     }
 
+    public static HealthStatus from(StreamInput in) throws IOException {
+        byte b = in.readByte();
```

Comment:
we can use `readEnum` and remove the `values` map

Reply:
done